warp10/src/main/java/io/warp10/worf/WorfCLI.java

```diff
@@ -161,6 +168,21 @@ public int execute(String[] args) {
           appName = cmd.getOptionValue(APPNAME);
         }
 
+        if (cmd.hasOption(LABELS)) try {
+            lbs = cmd.getOptionValue(LABELS);
+            String[] array = lbs.split(" ");
+            if (array.length % 2 != 0) {
+                throw new WorfException("Odd number of values provided as labels ");
+            }
+            for (int i = 0; i <= array.length - 1; i = +2) {
```

Comment:
Uses the ingress labels input format for Worf (xbeeId=XBee_40670F0D,moteId=53)
Spaces are allowed has value